doc/manual/modules.rst

```diff
@@ -290,12 +298,20 @@ initialization steps that must occur at *runtime* from steps that can
 occur at *compile time*.  For this purpose, Julia allows you to define
 an ``__init__()`` function in your module that executes any
 initialization steps that must occur at runtime.
+This function will not be called during compilation
+(``--output-*`` or ``__precompile__()``).
+You may, of course, call it manually if necessary,
+but the default is to assume this function deals with local state
+which does not need to be captured in the compiled image.
+And it will be called anytime the module is loaded,
```

Comment:
anytime, adv., adj., and int.
View as: Outline |Full entryKeywords: On |OffQuotations: Show all |Hide all
Pronunciation:  Brit.	 /ˈɛnɪtʌɪm/ , U.S.  /ˈɛniˌtaɪm/
Forms:  See any adj., pron., n., and adv.   and time n., int., and conj.
Frequency (in current use):  
Origin: Formed within English, by compounding. Etymons: any adj., time n.
Etymology: < any adj. + time n.
 
Compare sometime adv. and one time adv.
Often written as two words. (Word division in Middle English examples frequently reflects editorial choices of modern editors of texts, rather than the practice of the manuscripts.)
 
Compare adverbial use of the dative singular of the corresponding noun phrase in Old English, with both elements inflected for case and number:
OE   Wulfstan Institutes of Polity (Corpus Cambr. 201) 134   Nagan læwede men freolstidum ne fæstentidum þurh hæmedþingc wifes gemanan, þe ma þe heahhades menn þæt þing agon ænigum timan.
(Show Less)
 A. adv.
 1.
 
 a. Esp. with reference to a future event: at any point in time; (following a negative) ever.

a1375   William of Palerne (1867) l. 5148   Þat sche wold miswerche wrongli any time.
a1425   Wyclif Sel. Eng. Wks. (1871) II. 137   No jogelyng ne falseheed was ony tyme in Crist.
a1425   Rule St. Benet (Lansd.) (1902) 1 (MED),   Þat he..be not greuyd ony tyme be our euyl dedis.
c1450  (▸c1350)    Alexander & Dindimus (Bodl.) (1929) l. 484 (MED),   Whan þe watur wiþ þe wind þe wawus up casteth, And þouh hit turne any time to tempest of windus, Hit ne awecheþ no wawe nor no watur rereth.
a1538   A. Abell Roit or Quheill of Tyme f. 72v, in Dict. Older Sc. Tongue at Ony-time,   That nane suld dispair of the mercy of God onytyme.
1577   H. I. tr. H. Bullinger 50 Godlie Serm. II. To King sig. Aa.ij,   God..cannot any time either erre, or conceiue any false opinions or repugning counsells.
1650   T. Venner Treat. Tobacco in Via Recta (rev. ed.) 415   Tobacco any way, or any time used, is most pernicious unto dry melan-cholerick bodies.
1799   J. Robertson Gen. View Agric. Perth 433   If..the lease be forfeited or adjudged any time before the last years of the contract.
1859   ‘G. Eliot’ Adam Bede I. i. xiii. 251   It was as if she had been wooed by a river-god, who might any time take her to his wondrous halls below a watery heaven.
1932   W. Faulkner Light in August xv. 341   If you want to get to Jefferson anytime soon, you'd better go to town and hire a car.
1979   Observer 10 June 33/5   Suddenly you are in the Slide Area. No stopping any time.
2010   N.Y. Rev. Bks. 29 Apr. 29/3   Prospects for meaningful reforms anytime soon are surely limited.
(Hide quotations)
 
Categories »
 
 b. Followed by this or these and an expression of time: for the whole of the specified length of time up to the present. Cf. this pron. and adj. 5f, these pron. and adj. 3e. Now rare (arch. or regional in later use).

?1589   T. Nashe Almond for Parrat sig. D3,   You think that no man dare touch you, because you haue plaid the scuruie scolde anie time these twentie yeeres.
1592   G. Harvey Foure Lett. iii. 29   The poore Tennement of his Purse..hath bene the Diuels Dauncing schoole, anie time this halfe yeare.
1673   R. Leigh Transposer Rehears'd 39   This is a Plot..this has been a brewing any time this Thirty years.
1732   H. Baker & J. Miller tr. Molière Blunderer i. v. 23 in Sel. Comedies III   A Brace of Hundred Pieces that have been due any time this two long Years.
1879   H. James in Scribner's Monthly Dec. 209/2   He had been in love with Angela Vivian any time these three years.
1916   A. Lowell Men, Women & Ghosts 279   ‘I'll marry yer as soon As I c'n raise the money.’ ‘You've said that any time these five year, But you don't do nothin'.’
1955   F. Debenham Nyasaland 106,   I imagine that they could have retired to comfort and ease in England any time these last twenty years.
1989   S. Raven Blood of My Bone (2001) ii. 78   ‘When were you in a Greek prison, Julian?’..‘Any time these last fifteen years.’
(Hide quotations)
 
 
 2. In relative use: whenever, if ever.

1822   London Mag. Sept. 240   Here's my brother Felix can give me the breadth of my back five times out of four, any time he likes.
1873   Trollope Eustace Diamonds II. lii. 335   If I found I didn't like him, I'd leave him even after the altar. I'd leave him any time I found I didn't like him.
1915   P. G. Wodehouse Psmith, Journalist iv. 26   Any time you're in bad. Glad to be of service.
1976   National Observer (U.S.) 14 Aug. 4/1   The iron rule of business dictates staying open anytime there's a decent chance to turn a profit.
2011   D. Young Mastering Nikon D7000 iv. 219   Anytime you want to take pictures without a memory card, you'll need to select this setting.
(Hide quotations)
 
 B. adj. (attrib.).
 
  That can take place, be used, etc., at any time.

1870   A. Thackeray Let. Nov. in Lett. A. T. Ritchie (1924) viii. 143,   I did like your last letter. It was like a real anytime talk with you.
1904   Kingdom Jan. 1/1   Would it not be a good thing to meet the devil on his own ground and have an any time church?
1966   G. N. Leech Eng. in Advertising xv. 139   The any-time cereal.
2001   N.Y. Times 17 May g6/2   It is an anytime, anyplace solution.
(Hide quotations)
 
 C. int.
Categories »
 
  Orig. U.S. Used as a response to an expression of thanks. Cf. you are (or you're) welcome at welcome adj. 3d.

1961   Hartford (Connecticut) Courant 16 July 14 a/1   ‘Okay, thank you.’ ‘Any time.’
1985   M. Gagnon Inner Ring 37   ‘Thanks for the advice, Deirdre.’ ‘Any time,’ I said.
2001   M. Blake 24 Karat Schmooze xxv. 283   Reece said, ‘Cheers for seeing me right last night.’ ‘Anytime.’
(Hide quotations)
 
↑↓
Back to top

"anytime, adv., adj., and int." OED Online. Oxford University Press, June 2016. Web. 22 August 2016.